config/locales/devise.en-GB.yml

```diff
@@ -1,6 +1,6 @@
 # Additional translations at https://github.com/heartcombo/devise/wiki/I18n
 
-en:
+en-GB:
```

Comment:
I'm finding it hard to believe that this is right. Is there a reference somewhere? Surely en-GB should only apply to things with locale specialisation (like the spelling of specialisation) rather than to all English.

Reply:
I agree that it would be good if it would fall-back, however it didn't in my experimenting (I tried it without the rename first and all the tests broke). Suggestions / references welcome.